test/cadet/accounts/accounts_test.exs

```diff
@@ -1,30 +1,30 @@
 defmodule Cadet.AccountsTest do
   @moduledoc """
   Some tests in this module use pre-recorded HTTP responses saved by ExVCR.
-  this allows testing without the use of actual external IVLE API calls.
+  this allows testing without the use of actual external LumiNUS API calls.
 
   In the case that you need to change the recorded responses, you will need
-  to set the config variables `:ivle_key` (used as a module attribute in
-  `Cadet.Accounts.IVLE`) and environment variable TOKEN (used here). Don't
+  to set all the luminus config vars (used as a module attribute in
+  `Cadet.Accounts.Luminus`) and environment variable CODE (used here). Don't
   forget to delete the cassette files, otherwise ExVCR will not override the
   cassettes. You can set the TOKEN environment variable like so,
 
-    TOKEN=very_long_token_here mix test
+    CODE=auth_code_goes_here mix test
 
-  Token refers to the user's authentication token. Please see the IVLE API docs:
-  https://wiki.nus.edu.sg/display/ivlelapi/Getting+Started
-  To quickly obtain a token, simply supply a dummy url to a login call:
-      https://ivle.nus.edu.sg/api/login/?apikey=YOUR_API_KEY&url=http://localhost
-  then copy down the token from your browser's address bar.
+  Code refers to the authorization code generated via the OAuth Authorization
+  Grant Type. More information can be found here
+
+  https://wiki.nus.edu.sg/pages/viewpage.action?pageId=235638755.
   """
 
   use Cadet.DataCase
   use ExVCR.Mock, adapter: ExVCR.Adapter.Hackney
 
   alias Cadet.{Accounts, Repo}
-  alias Cadet.Accounts.{Query, User}
+  alias Cadet.Accounts.{Query, Luminus, User}
 
-  @token if System.get_env("TOKEN"), do: System.get_env("TOKEN"), else: "token"
+  @code System.get_env("CODE") || nil
```

Comment:
```suggestion
  @code System.get_env("CODE")
```
`System.get_env/1` would already return `nil` if the environment variable is undefined